pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1233,9 +1224,6 @@ self: super: {
   # dontCheck can be removed on the next package set bump
   vulkan-utils = dontCheck (addExtraLibrary pkgs.vulkan-headers super.vulkan-utils);
 
-  # https://github.com/dmwit/encoding/pull/3
-  encoding = doJailbreak (appendPatch ./patches/encoding-Cabal-2.0.patch super.encoding);
```

Comment:
This one is *slow* to build, sat for a few minutes during "processing library" stage for no apparent reason.